src/js/profile/mobile/widget/TextInput.js

```diff
@@ -254,6 +260,18 @@
 					 * @member ns.widget.mobile.TextInput
 					 */
 					ERROR: "ui-error",
+					/**
+					 * Set class for icon assigned to text input
+					 * @style ui-text-input-icon
+					 * @member ns.widget.mobile.TextInput
+					 */
+					ICON: CLASSES_PREFIX + "-icon",
+					/**
+					 * Set class for icon assigned to text input when TextInput is not empty and valid
+					 * @style ui-text-input-not-empty
+					 * @member ns.widget.mobile.TextInput
```

Comment:
It seems that the selector is different from the defined style.

Reply:
Done